lib/MusicBrainz/Server/Report/RecordingsSameNameDifferentArtistsSameName.pm

```diff
@@ -4,6 +4,9 @@ use Moose;
 with 'MusicBrainz::Server::Report::RecordingReport',
      'MusicBrainz::Server::Report::FilterForEditor::RecordingID';
 
+# FIXME: This report has been disabled since the upgrade to PG 12, because
+# its query can no longer execute in under 5 minutes in production.
```

Comment:
```suggestion
# MBS-10843: This report has been disabled since the upgrade to PG 12,
# because its query can no longer execute in under 5 minutes in production.
```